labs/iot/gps-lab/steps/add-gps-to-pi.md

```diff
@@ -0,0 +1,19 @@
+# Add the GPS sensor to Raspberry Pi
+
+In this step you will set up the GPS sensor to Pi.
+
+## Steps
+
+1. Put the vcc to 2, ground to 6 tx and rx to 8 and 10
+![Pin Reference](../images/raspberry-pi-pins.jpg)
+2. The GPS Module connected should look like following with a blinking red light when the satelite is available
```

Comment:
How long till the satellite is available? What happens if the light doesn't blink? Any tips on placement?